src/java/com/wolfssl/provider/jsse/WolfSSLUtil.java

```diff
@@ -87,19 +87,27 @@ protected static String[] sanitizeProtocols(String[] protocols,
         WolfSSLDebug.log(WolfSSLUtil.class, WolfSSLDebug.INFO,
             () -> "jdk.tls.disabledAlgorithms: " + tmpDisabledAlgos);
 
+        /* 
+         * WolfJSSE only supports DTLSv1.3, automatically add DTLSv1,
+         * and DTLSv1.2 to disabled algorithms for now */
+
+        disabledAlgos += ",DTLSv1,DTLSv1.2";
+
         /* If WolfSSL.INVALID is passed in as currentVersion, no filtering
          * is done based on current protocol */
         if (currentVersion != WolfSSL.TLS_VERSION.INVALID) {
             /* Remove DTLS protocols if using TLS explicitly. Needed
              * since native wolfSSL doesn't have protocol masks for DTLS. */
-            if (currentVersion != WolfSSL.TLS_VERSION.DTLSv1_2) {
-                disabledAlgos += ",DTLSv1.2";
-            }
             if (currentVersion != WolfSSL.TLS_VERSION.DTLSv1_3) {
                 disabledAlgos += ",DTLSv1.3";
             }
         }
 
+        final String disabledAlgs = disabledAlgos;
+        WolfSSLDebug.log(WolfSSLUtil.class, WolfSSLDebug.INFO,
```

Comment:
I don't think this print statement quite makes sense.  We don't have a "wolfjsse.disabledProtocols" property, which this is indicating we do.  I think we can probably remove this.